include/swift/AST/DiagnosticsSema.def

```diff
@@ -1973,7 +1973,7 @@ WARNING(protocol_usable_from_inline_warn,none,
         "protocol %select{refined|used}0 by '@usableFromInline' protocol "
         "should be '@usableForInline' or public", (bool))
 ERROR(protocol_property_must_be_computed_var,none,
-      "immutable property requirement must be declared as 'var' with a "
+      "protocols cannot require properties to be immutable; declare read-only properties by using 'var' with a "
```

Comment:
Could you please add some tests for this (or update the existing ones if you haven’t)?

Reply:
@theblixguy I updated the test files. Can you check now if everything is okay?

Reply:
Looks great! Thank you!